vercel.json

```diff
@@ -6,11 +6,44 @@
       "shoppy-metabot": true
     }
   },
+  "headers": [
```

Comment:
caching static assets

Reply:
By much you mean the duration of the cache, or the way we use public image?

We could adjust the duration of the cache. We could import them via code yes, but I don't want this PR to be too big. Also, some folder like `images` are saved in the db as path, so we still need to cache those files. IIRC, we return the product alongside its image URL like `/images/product1.png` or something like that.

Reply:
Yes that's good enough, it was just a year seemed a bit too much in case we'll change them and people would have the old ones after a year.
If we change the images we should rename them anyway though so 🤷